ngi_reports/common/__init__.py

```diff
@@ -29,6 +30,26 @@ def __init__(self, config, LOG, working_dir, **kwargs):
     def parse_piper_xml(self):
         """ Parses the XML setup files that Piper uses
         """
+        
+        def _nextitem(xml):
```

Comment:
Much nicer :+1: My implementation made me squirm a little, I figured that there should be a better way to do it..